src/app/content/components/BookBanner.tsx

```diff
@@ -213,7 +213,7 @@ const BookBanner = () => {
     }
     e.preventDefault();
 
-    if (!await showConfirmation(services)) {
+    if (!await showConfirmation(intl)) {
```

Comment:
this is gonna conflict with bartek's thing, leave it as services

Reply:
does it need services other than intl? can I pass them in separately since we're talking about removing intl from services?

Reply:
nvm, saw your other comment